zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs

```diff
@@ -391,6 +391,24 @@ pub async fn test_responses<State, ReadState>(
         .expect("We should have a validate_address::Response");
     snapshot_rpc_validateaddress("invalid", validate_address, &settings);
 
+    // `z_validateaddress`
+    let founder_address = match network {
```

Comment:
Optional: Can you please add a p2pkh and sapling address, and the unified test vectors from z_listunifiedreceivers to the snapshot tests?
Then we'll have full coverage of all the address types.

It's ok to use the same p2pkh and sapling addresses as the ones inside those unified addresses. We don't need to cover both networks for all of these.

Sorry I didn't see this before.